### PR TITLE
Update apollo-server to version 3 and graphql to version 16

### PR DESCRIPTION
**Pull request by lukasbals**
With this change, it's possible to use the `apollo-server-integration-testing` with the latest version of `apollo-server-express@^3.0.0` and `graphql@^16.0.0`.

### CODE REVIEW

**Review by gustawdaniel — APPROVED** [review]

**Comment by lukasbals** [comment]
Can someone approve the workflow? Th PR is approved but the worklow didn't run.
@gustawdaniel can you approve the workflow?

**Comment by dchambers** [comment]
Thanks for doing this @lukasbals 🙏 . Any chance this could be moved along please @gustawdaniel?

**Comment by vitorbal** [comment]
@msholty-fd do you know who could help move this along? I unfortunately don't have maintainer rights anymore

**Comment by Berger92** [comment]
This is really helpful @lukasbals 🙇 
Could we move forward with this @gustawdaniel, please 🙏 ?

**Comment by saegeullee** [comment]
I need this to be merged. please check this out and merge @gustawdaniel

**Review by filipe1309 — APPROVED** [review]

**Comment by lukasbals** [comment]
Thanks for the approvals! But we need approval from someone with write access to merge. @filipe1309  or @gustawdaniel do you know somone having write access?

**Comment by jodok** [comment]
please can we get this merged?

**Comment by vitorbal** [comment]
I reached out to some ex-colleagues at Zapier, let me see if I can get someone to take a look.

**Review by msholty-fd — APPROVED** [review]

**Comment by adijesori** [comment]
Hello @lukasbals @msholty-fd Any idea how can I use this fix? I believe this fix is not yet available through npm

**Pull request merged**
